Jellyfin.NfoMetadata/Providers/AlbumNfoProvider.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.IO;
+using Jellyfin.NfoMetadata.Models;
+using MediaBrowser.Controller.Entities.Audio;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.IO;
+using MediaBrowser.Model.Serialization;
+using Microsoft.Extensions.Logging;
+
+namespace Jellyfin.NfoMetadata.Providers
+{
+    /// <summary>
+    /// The music album nfo metadata provider.
+    /// </summary>
+    public class AlbumNfoProvider : BaseNfoProvider<MusicAlbum, AlbumNfo>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AlbumNfoProvider"/> class.
+        /// </summary>
+        /// <param name="logger">Instance of the <see cref="ILogger{TCategoryName}"/> interface.</param>
+        /// <param name="fileSystem">Instance of the <see cref="IFileSystem"/> interface.</param>
+        /// <param name="xmlSerializer">Instance of the <see cref="IXmlSerializer"/> interface.</param>
+        public AlbumNfoProvider(ILogger<BaseNfoProvider<MusicAlbum, AlbumNfo>> logger, IFileSystem fileSystem, IXmlSerializer xmlSerializer)
```

Comment:
```suggestion
        public AlbumNfoProvider(ILogger<AlbumNfoProvider> logger, IFileSystem fileSystem, IXmlSerializer xmlSerializer)
```

Same for all providers.
Will have to change in tests as well.